tests/console/lvm.pm

```diff
@@ -123,4 +128,23 @@ sub run {
     assert_script_run 'pvdisplay';
 }
 
+# Checks if lvmetad reads devices/filter (bsc #1163526)
+sub check_lvmetad_filter {
+    for (my $i = 1; $i <= 2; $i++) {
+        assert_script_run 'dd if=/dev/zero of=disk_' . $i . '.img bs=1M count=32 status=none';
+        assert_script_run 'losetup /dev/loop' . $i . ' disk_' . $i . '.img';
+        assert_script_run 'pvcreate /dev/loop' . $i;
+    }
+    assert_script_run 'losetup /dev/loop3 disk_2.img';
```

Comment:
Why is this here after the loop above ?

Reply:
TW test is passing not even noticing /dev/loop3, and 15sp1 is failing  on duplicate device.
That's some inconsistent behaviour or bug ?

Reply:
> Why is this here after the loop above ?

The 3rd losetup after the loop is there to attach a new loop device to disk_2.img, where a loop device is already attached to.

>TW test is passing not even noticing /dev/loop3, and 15sp1 is failing on duplicate device.
>That's some inconsistent behaviour or bug ?

https://bugzilla.suse.com/show_bug.cgi?id=1163526
It is a bug which appears in SLE but not tumbleweed, which is why I included the tumbleweed run. The inconsistency lies in the fact that --cache --config is passed my $filter, which removes the conflicting loop3 device, however the filter is not read in the newest lvm2 updates.

Reply:
Can you add workaround for bsc#1163526 ?

Reply:
Added soft fail